android/app/src/main/java/com/tuneify/MainApplication.kt

```diff
@@ -9,8 +9,8 @@ import com.facebook.react.ReactPackage
 import com.facebook.react.defaults.DefaultNewArchitectureEntryPoint.load
 import com.facebook.react.defaults.DefaultReactHost.getDefaultReactHost
 import com.facebook.react.defaults.DefaultReactNativeHost
-import com.facebook.react.soloader.OpenSourceMergedSoMapping
 import com.facebook.soloader.SoLoader
+import com.tuneify.BuildConfig
```

Comment:
explain the reason for this?

Reply:
while building its android app
i was encountering error of cmake build failed and and debug kotlin failed somewhat similar to this
`[Unresolved reference: R and BuildConfig in React Native Android Project with Kotlin]`

so i opened the android folder in android studio,
i saw build config was user 4 times (line: 27, 29, 30, 39) in this file but it was not available, It was suggesting to import it

so this is the reason

Reply:
is it because of the RN version??